src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Common/ProjectPackageCollections.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
```

Comment:
The feedback I provided for [`NuGetProjectPackages`](https://github.com/NuGet/NuGet.Client/pull/3680/files#diff-fe000637066266aad1cc250ab6f4c870d21df03b6d11b171fbe5b17d7fa3b569) applies here as well.

Reply:
Fixed